airflow/dags/mental_health_dag.py

```diff
@@ -0,0 +1,129 @@
+import json
+import logging
+
+import pendulum
+import pymsteams
+import requests
+from airflow import DAG
+from airflow.hooks.base import BaseHook
+from airflow.models import Variable
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger("ContentCreationLogger")
+
+
+def load_quote(ti):
+    """Task to load a quote"""
+    api_url = Variable.get("QUOTE_API_URL")
+    try:
+        response = requests.get(api_url)
+        response.raise_for_status()
+        quote_data = response.json()
+        quote = quote_data.get("content")
+        author = quote_data.get("author")
+        if quote and author:
+            logger.info("Quote is successfully loaded")
+            ti.xcom_push(key="quote", value=f"{quote} — {author}")
```

Comment:
I guess it's better to push them as separate values and then combine into required format during sending - you never know how your data would be used, and it might be hard to later separate those values